core/bin/external_node/src/main.rs

```diff
@@ -279,11 +280,41 @@ async fn init_tasks(
             .with_tx_sender(tx_sender.clone(), vm_barrier.clone())
             .with_sync_state(sync_state.clone())
             .enable_api_namespaces(config.optional.api_namespaces())
-            .build(stop_receiver.clone())
+            .build(stop_receiver.clone(), APIMode::Modern)
             .await
             .context("Failed initializing HTTP JSON-RPC server")?;
 
+    let legacy_http_server_handles =
+        ApiBuilder::jsonrpsee_backend(config.clone().into(), connection_pool.clone())
```

Comment:
- `ApiBuilder` may be made `Clone` to avoid this copy-paste.
- Dumb question: Do we *want* to run 2 instances of each API server? IMO, it makes little sense; instead, `ApiMode` should be a config param, and each server should run as a single instance taking this param into account.

Reply:
I guess it makes sense